web/src/beta/features/ProjectSettings/innerPages/StorySettings/index.tsx

```diff
@@ -54,7 +54,10 @@ const StorySettings: React.FC<Props> = ({
         ) : (
           <StorySettingsDetail
             key={currentStory.id}
-            settingsItem={currentStory}
+            settingsItem={{
+              ...currentStory,
+              bgColor: currentStory.bgColor as string,
```

Comment:
I feel like if this is necessary typings are wrong somewhere 🤔

Reply:
I added this typing because I faced this error "Type 'Maybe<string> | undefined' is not assignable to type 'string | undefined'.
  Type 'null' is not assignable to type 'string | undefined'" so also to make sure the bgColor has default value , do you have any suggestion here ?

Reply:
If you are getting `Type 'null' is not ...` errors that means you need to either add it to the type, or do something like `someValue || undefined` so that if someValue is NULL it'll become undefined, not NULL